src/client/components/ProfileFormComponent/ProfileFormComponent.stories.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Avatar from './ProfileFormComponent';
+import PropTypes from 'prop-types';
+
+export default {
+  title: 'Avatar',
+  component: Avatar,
+  argTypes: {
+    color: {
+      control: { type: 'color' },
```

Comment:
I guess, this control is supposed to be select rather than type:color. As the docs says, there would be 3 options of colors only not the whole spectrum :) 

> COLORS #DB643D, #535C75 and # 000000

Reply:
yes i changed the color type to select.